src/server.js

```diff
@@ -132,6 +132,56 @@ app.get('/edit', checkLogin, (req, res) => {
   res.send(_renderDocument({ title: '[WIP] Edit', locale, content }))
 })
 
+app.get('/edit/:id(name|address|maritalStatus|children)?', checkLogin, (req, res) => {
```

Comment:
We might want to add a comment pointing to https://expressjs.com/en/guide/routing.html and/or https://www.npmjs.com/package/path-to-regexp just so it's easier to know how this works